ansys/mapdl/core/_commands/parse.py

```diff
@@ -99,3 +99,9 @@ def parse_output_volume_area(msg):
         res = re.search(r"OUTPUT (AREA|VOLUME|AREAS) =\s*([0-9]+)", msg)
         if res is not None:
             return int(res.group(2))
+
+def parse_ndist(msg):
+    """Parse the node value from a node message"""
+    finds = re.findall(NUM_PATTERN, msg)[-4:]
+    if len(finds) == 4:
+        return [float(val) for val in finds]
```

Comment:
Should have an additional blank line at the end of the file. See:
https://www.flake8rules.com/rules/W391.html

Reply:
Thank you, Alex(@akaszynski),
I have changed.